modules/guides/css/guides.in-page-navigation.css

```
@@ -0,0 +1,69 @@
+#guides__flex-wrapper{
+  min-width:20%;
+  margin-right:1rem;
+}
+
+.region-content {
+  display: flex;
+}
+
+#guides__in-page-nav.guides__in-page-nav--sticky {
+  position: fixed;
+  top: 0;
+  margin-top: 80px;
+}
+
+#guides__in-page-nav,
+#guides__in-page-nav ul,
+#guides__in-page-nav li {
+  margin: 0px;
+  padding: 0px;
+  text-indent: 0px;
+  list-style-type: none;
+}
+
+#guides__in-page-nav {
+  border-top: 1px solid #ddd;
+}
+
+#guides__in-page-nav .guides__in-page-nav-li {
```

Same here, don't need this specificity.

```suggestion
.guides__in-page-nav-li {
```